MaintenanceManager/MaintenanceManager.json

```diff
@@ -87,12 +87,12 @@
             }
         },
         "setMaintenanceMode":{
-            "summary": "Sets the maintenance mode",
+            "summary": "Sets the maintenance mode. \n \n### Events\n \n No Events.",
             "params": {
                 "type":"object",
                 "properties": {
                     "maintenanceMode":{
-                        "summary": "The maintenance mode. The `FOREGROUND` mode runs all maintenance tasks. The `BACKGROUND` mode runs maintenance tasks that do not impact the user experience.",
+                        "summary": "The maintenance mode. The `FOREGROUND` mode runs all maintenance tasks in the foreground. The `BACKGROUND` mode runs maintenance tasks in the background that do not impact the user experience.",
```

Comment:
The `FOREGROUND` mode runs all maintenance tasks.
BACKGROUND-
    Abort activities currently running and if the task can't run in the background (if maintenance was already started with FOREGROUND mode) 
    Execute tasks not impacting user experience if the maintenance mode is set to BACKGROUND before calling startMaintenance

Reply:
Added the given description